### DISCUSSION TITLE
View the zone in VEDO

### DISCUSSION THREAD

**Discussion by alex81go** [gicamm/homeassistant-comelit#105] [General]
I can't see the status of the sensors (zones).
The only data I see is the status of the areas.

**Comment by gicamm** [comment]
It's usually helpful to provide more details about the issues encountered; otherwise, it can be challenging to understand what's happening and how to assist effectively.

**Reply by gianlucaruggeri1976** [reply]
Ciao Giovanni,
scusa il disturbo: secondo te ci sono possibilità per risolvere questo problema?
Grazie

**Comment by alex81go** [comment]
![Immagine 2024-11-07 143642](https://github.com/user-attachments/assets/1c9e2749-1d5b-4144-9ba2-d3921b82303f)
![Immagine 2024-11-07 143922](https://github.com/user-attachments/assets/f852dfec-c706-4b5e-9f3a-98025a8a4950)

I entered the code on configuration.yaml with all the required parameters. Installed the Comelit integration via HACS and rebooted.
In the entities only the 8 areas are displayed with their respective names, but I can't see the status of the individual zones.

Thanks for the help

**Comment by gianlucaruggeri1976** [comment]
@gicamm Ciao Giovanni, 
innanzitutto ti ringrazio per il prezioso supporto.
Anch'io ho lo stesso problema: vedo solo le entità (termostati e attivazioni allarme) ma non lo stato dei vari sensori
Come potrei risolvere?
Grazie
Ciao
![Screenshot 2025-10-14 133036](https://github.com/user-attachments/assets/55be8afd-0a0e-4063-93ed-8103fad35df3)
